kinto/authorization.py

```diff
@@ -25,7 +25,9 @@
 # Ex: bucket:read is granted by both bucket:write and bucket:read
 PERMISSIONS_INHERITANCE_TREE = {
     '': {  # Granted via settings only.
-        'bucket:create': {},
+        'bucket:create': {
+            '': ['bucket:create', 'create']
```

Comment:
It doesn't makes sense to inherit `bucket:create` from `bucket:create` because we already have it.

Reply:
No, I don't think this `PERMISSIONS_INHERITANCE_TREE` dict should be touched in this PR. 
This dict gives the permission inheritance, and there is permission that is above `bucket:create` since it can be given via settings.

However in the permissions endpoint code, we can manipulate the `perms_descending_tree` and adjust the `from_settings()` function.

Reply:
Oh this is because we don't inherit settings permission just yet ok...